### PR TITLE
Change batchSize to batchPerThread for `predict` API

### PR DESCRIPTION
**Pull request by zhichao-li**

### CODE REVIEW

**Comment by zhichao-li** [comment]
@hhbyyh  Seems like nnframe only depend on `forward` method only, so I suppose this changes would not affect `nnframe` API.

**Comment by zhichao-li** [comment]
Jenkins passed.

File: pyzoo/zoo/pipeline/api/keras/engine/topology.py [hunk 1]
```diff
@@ -207,25 +207,23 @@ def convert_output(output):
...
         # Arguments
         x: Prediction data. A Numpy array or RDD of Sample or ImageSet.
-        batch_size: When distributed is True, the default value is 4 * rdd.getNumPartitions.
-                    When distributed is False the default value is 4 * numOfCores.
-                    If you want to tune the batch_size,
-                    you should make sure its value can be divisible by
-                    rdd.getNumPartitions(distributed=True) or numOfCores(distributed=False)
+        batch_per_partition:
+          When distributed is True,the default total batch size is 4 * rdd.getNumPartitions.
+          When distributed is False the default total batch size is 4 * numOfCores.
```
Comment (hhbyyh) [inline pyzoo/zoo/pipeline/api/keras/engine/topology.py]: 4 can be misleading here. does it mean `batch_per_partition` * `rdd.getNumPartitions`.

And for local mode where x is nparray, I don't think `numOfCores` is corresponding to `rdd.getNumPartitions`. Just using `batch_per_partition` as the batch size for local mode seems to be more intuitive for me.

Comment (zhichao-li) [reply pyzoo/zoo/pipeline/api/keras/engine/topology.py]: Currently, we  try to split the input data by numOfCores for processing the data in parallel, so the effective total_batch_size still batch_per_partition * numOfCores.
Maybe change the `batchPerPartition` to `batchPerThread` would be more straightforward.

**Comment by zhichao-li** [comment]
ZOO-PR-Validation/906/

**Pull request merged**